explain geo_point mapping in geoip-processor (#29114)
elastic/elasticsearch: docs/plugins/ingest-geoip.asciidoc

[source,js]
--------------------------------------------------
{
  "_index" : "my_index",
  "_type" : "_doc",
  "_id" : "my_id",
  "_version" : 1,
  "found" : true,
  "_source" : {
    "ip" : "80.231.5.0"
  }
}
--------------------------------------------------
// TESTRESPONSE

[[mappings-note]]
===== Recognizing Location as a Geopoint
Although this plugin enriches your document with a `location` field containing

> the estimated latitude and longitude of the ip-address, this field will not be

Reviewer: "ip-address" -> "IP address"

indexed as a <<geo-point,`geo_point`>> type in Elasticsearch without explicitely defining it
as such in the mapping.

> For example, here would be the mapping for the example index used in the above examples

Reviewer: How about being more direct? 
```suggestion
You can use the following mapping for the example index above:
```

--------------------------------------------------
// TESTRESPONSE

[[mappings-note]]
===== Recognizing Location as a Geopoint
Although this plugin enriches your document with a `location` field containing
the estimated latitude and longitude of the ip-address, this field will not be
indexed as a <<geo-point,`geo_point`>> type in Elasticsearch without explicitely defining it
as such in the mapping.

For example, here would be the mapping for the example index used in the above examples

[source,js]
--------------------------------------------------
{
  "mappings": {
    "my_type": {
      "properties": {
        "geoip": {
          "properties": {
            "location": { "type": "geo_point" }
          }
        }
      }
    }
  }
}
--------------------------------------------------

> // NOTCONSOLE

Reviewer 2: Can you make the example code a request to set mappings or create an index so this is tested?

Author: you make a reasonable request — will update.
